### PR TITLE
(Updated) discord.js#internal-sharding support added

### PR DESCRIPTION
**Pull request by jas777**
Okay, I've corrected pull request to match current API structure.

**NOTE** This will _not_ work with normal d.js

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('./src/index');
+module.exports = require('./src/index');
```
Comment (tonkku107) [inline index.js:1]: ??

File: src/index.js [hunk 2]
```diff
@@ -145,11 +145,8 @@ class DBLAPI extends EventEmitter {
       data.shard_count = shardCount;
     } else {
       data.server_count = this.client.guilds.size;
-      if (this.client.shard && this.client.shard.count) {
```
Comment (tonkku107) [inline src/index.js:148]: You should do this without removing support for normal discord.js

**Review by tonkku107 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (jas777) [reply src/index.js:148]: But normal d.js doesn't have client.ws, ofc you can check if(client.ws.shards), but I did it late night and to be honest I made it just to make it. Feel free to adjust it.

File: index.js (hunk 1 above)
Comment (jas777) [reply index.js:1]: GitHub bug

File: src/index.js (hunk 2 above)
Comment (tonkku107) [reply src/index.js:148]: It's quite literally as simple as bring back the code you removed and keep the new code

**Pull request closed without merging**